WolvenKit.RED4.TweakDB/FlatsPool.cs

```diff
@@ -48,6 +48,7 @@ public void Add(string flat, IType value)
                 }
 
                 _flats.Add(flat, valueHash);
+                Console.WriteLine($"Adding flat: {flat}: {value}");
```

Comment:
This should not be here, we do not have a logger implemented in this project, but might be worth doing it.

Reply:
There's `ILoggerService _loggerService` elsewhere, but I'm not sure how to get it at this level - is that something we'd want to do? I found knowing the flats & records that were being created useful, especially since there's the appending that happens with records.

Reply:
the ILoggerService lives in WolvenKit.Core (so it is accessible from .TWeakDB) and is implemented through dependency injection in the front-ends (wolvenkit.WPF and wolvenkit.CLI). 
It would need to be dependency-injected and teh tweakDB registered in the IoC container. @WopsS any thoughts?